students/Roy_Tate/lesson04/mailroom2.py

```diff
@@ -0,0 +1,184 @@
+# author/student: Roy Tate (githubtater)
+
+from collections import Counter
+
+import sys
+import os
+import os.path
+
+# Create the donor_dict and manually add a few entries
+donor_dict = {}
```

Comment:
It's more conventional in Python to define a dict using literal syntax:
```
donor_dict = {
    'Bill': [1100, 2100, 3100],
    'Mark': [1000, 300000.23, 2000],
}
```
The syntax you use below is generally used for adding keys/values to an existing dict.